dbms/src/Common/RandomData.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Common/RandomData.h>
+#include <fmt/format.h>
+
+#include <random>
+
+namespace DB::random
+{
+
+String randomString(UInt64 length)
+{
+    static const std::string charset{
+        "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz!@#$%^&*()|[]{}:;',<.>`~"};
+    std::random_device rand_dev;
+    std::mt19937_64 rand_gen(rand_dev());
+    String str(length, 0);
```

Comment:
```suggestion
    String str(length, '\x00');
```

this is more explicit